README.md

```diff
@@ -109,6 +109,55 @@ A glossary entry is structured like this:
     should `define('something')` in RStudio put the definition in the help pane
     (and if so, should it hyperlink to terms that the definition depends on)?
 
+## Use Cases
+
+1.  **Linking to a definition.**
+    1.  *Amari* writes a lesson in R Markdown that introduces some new terms.
+    1.  She has defined the language to be Spanish using the `glossary/language` key
```

Comment:
Clarification question: Does `glossary/language` mean that the yaml header will have:

```yaml
glossary:
  - language: es
```

Reply:
:+1: If there's a cleaner way to write nested YAML keys inline, please let me know - path notation was the only thing I could think of.

Reply:
Yes @zkamvar that would be the structure. Closing this comment to merge the PR.